### PR TITLE
Fix consumer config

### PR DESCRIPTION
**Pull request by yuriysemen-edvantis**
Made consumer to use topic from configuration.

### CODE REVIEW

**Comment by yuriysemen-edvantis** [comment]
@morissette Could You please review this and apply?

**Comment by morissette** [comment]
See latest version; and let me know what changes need to be made.

File: eventify/consumer.py [hunk 1]
```diff
@@ -14,7 +14,9 @@ def handler(count):
             print("event received: {0}", count)
 
         try:
-            yield self.subscribe(handler, u'mytopic')
+            topic = self.config.extra['publish_topic']['topic']
```
Comment (morissette) [inline eventify/consumer.py:17]: yeah these changes have already been committed - my apologies on not pushing earlier

**Comment by morissette** [comment]
Closing as changes in PR have already been implemented.

**Pull request closed without merging**